### PR TITLE
Added update profile

### PR DESCRIPTION
**Pull request by andresg4**
## TASKS
- [X] Added update method to users controller.
- [X] Added update route.

### CODE REVIEW

File: app/controllers/api/v1/users_controller.rb [hunk 1]
```diff
@@ -1,11 +1,37 @@
...
   module V1
     class UsersController < ApplicationController
-      before_action :authenticate_user!, only: [:targets]
+      before_action :authenticate_user!, only: %i[update targets]
+
+      def show; end
+
+      def update
+        user.update!(user_params)
+        render partial: 'api/v1/users/show', locals: { user: current_user }, status: :ok
```
Comment (Alxzu) [inline app/controllers/api/v1/users_controller.rb]: Can you try without `api/v1/users/`since you are already on that namespace

File: app/controllers/api/v1/users_controller.rb [hunk 2]
```diff
@@ -1,11 +1,37 @@
...
+      before_action :authenticate_user!, only: %i[update targets]
+
+      def show; end
+
+      def update
+        user.update!(user_params)
+        render partial: 'show', locals: { user: current_user }, status: :ok
+      end
 
       def targets
```
Comment (jdrosales17) [inline app/controllers/api/v1/users_controller.rb]: I would remove this action from this controller and use the same logic on the `index` action of the targets controller, where you should return only the targets of the current user instead of all the targets. This is not part of the PR but it is worth changing (remember to also remove the related route and view)

**Review by jdrosales17 — APPROVED** [review]

**Review by Alxzu — APPROVED** [review]
